frontend/types/api/validator_dashboard.ts

```diff
@@ -75,7 +76,8 @@ export interface VDBGroupSummaryData {
   sync: VDBGroupSummaryColumnItem;
   sync_count: VDBGroupSummarySyncCount;
   slashings: VDBGroupSummaryColumnItem; // Failed slashings are count of validators in the group that were slashed
-  proposal_validators: number /* uint64 */[];
+  proposal_validators: number /* uint64 */[]; // fill with up to 3 validator indexes
+  proposal_validator_count: number /* uint64 */; // number of distinct validators
```

Comment:
This will contain the proposal validators count. Keep in mind that, unlike sync and slash, this does not get a `VDBGroupSummaryColumnItem`, since proposal data is already available earlier in the table call, as to avoid redundant data.